doc/topics/the-event-loop-timers-and-nexttick.md

```diff
@@ -0,0 +1,373 @@
+# The Node.js Event Loop, Timers, and `process.nextTick()`
+
+## What is the Event Loop?
+
+The event loop is what allows Node.js to perform non-blocking I/O
+operations — despite the fact that JavaScript is single-threaded — by
+offloading operations to the system kernel whenever possible. 
+
+Since most modern kernels are multi-threaded, they can handle
+multiple operations executing in the background. When one of these
+operations completes, the kernel tells Node.js so that the appropriate callback 
+may added to the `poll` queue to eventually be executed. We'll explain
+this in further detail later in this topic. 
+
+## Event Loop Explained 
+
+When Node.js starts, it initializes the event loop, processes the provided
+input script (or drops into the REPL, which is not covered in this document)
+which may make async API calls, schedule timers, or call `process.nextTick()`, 
+then begins processing the event loop. 
+
+The following diagram shows a simplified overview of the event loop's
+order of operations.
+
+       ┌───────────────────────┐
```

Comment:
I consider them show-stoppers.

The order of steps is highly important, because otherwise some behaviors cannot be explained, creating more confusion.

IMHO it is acceptable to simplify things in a presentation (you have a limited time), but this should be the 'ultimate' docs on the event loop, so nobody would have to dig through all the source to understand how things works. So, let's aim for correctness first, and then prepare a "newbie friendly"  doc after (or leave it to 3rd-party blog posts).